install.sh

```diff
@@ -0,0 +1,98 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Not SUSE operating system."
+        return -1
+    fi
+    case "$VERSION_ID" in
+    "15.3")
```

Comment:
Are we sure we only want to support this version?
I would say that the majority of our users will have older versions

Reply:
The older versions will come later. There is go1.16 and hence the rpm only in 15.3.

Reply:
Changed.

Reply:
We need to make this installer immediately useful for as many people as possible.
Maybe we could download the pre-compiled binary from GitHub for all cases where the rpm is not available?

Reply:
I think the existing rpm should work as well for older sle 15.x releases, I will check it.

Reply:
> We need to make this installer immediately useful for as many people as possible.
> Maybe we could download the pre-compiled binary from GitHub for all cases where the rpm is not available?

It actually works on sle 15.0 just as file. So I will relax the OS release requirements.